src/components/Donation/__tests__/MembershipWidget.spec.tsx

```diff
@@ -189,7 +189,7 @@ test('allows to complete flow using an amount donation selection', async () => {
     api: {
       // TODO: check on integration time seems to be something wrong
```

Comment:
I feel this is related to the fact that I didn't add the `errors` properly but in the rush to deliver the first cycle I just add the TODO. **Can we remove this TODO? And the one in line 105**